target/linux/lantiq/dts/DM200.dts

```diff
@@ -0,0 +1,204 @@
+/dts-v1/;
+
+#include "vr9.dtsi"
+
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "DM200 - Netgear DM200";
+
+	chosen {
+		bootargs = "console=ttyLTQ0,115200";
+	};
+
+	aliases {
+		led-boot = &power_green;
+		led-failsafe = &power_amber;
+		led-running = &power_green;
+
+		led-internet = &dsl_green;
```

Comment:
I've pushed a commit a few minutes ago, which required a change here. Please use the following:

```
led-dsl = &dsl_green;
```